src/modules/auth/auth.service.ts

```diff
@@ -19,54 +24,37 @@ export class AuthService {
     const user = await this.authRepo.findOneBy({ email });
 
     if (user) {
-      throw new HttpException(
-        {
-          status: HttpStatus.BAD_REQUEST,
-          error: 'This email already exist',
-        },
-        HttpStatus.BAD_REQUEST,
-      );
-    } else {
-      const newUser = {
-        email: email,
-        password: password,
-      };
+      throw new ConflictException('User already exists');
+    }
 
-      const salt = genSaltSync(10);
-      newUser.password = hashSync(newUser.password, salt);
-      const createdUser = await this.authRepo.save(newUser);
+    const salt = await genSalt(10);
```

Comment:
magic number?) bad practice. move it to the constant with name that will explain what is it

Reply:
True, I was so used to it that I didn't even notice :sweat_smile:. I'll fix it